### PR TITLE
Convert localized orders values to config

### PR DESCRIPTION
**Pull request by dcsaszar**
https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-localize-order.scrivito-portal-app.pages.dev/orders?_scrivito_workspace_id=i6c8d84ddc5950c2&_scrivito_display_mode=diff

(force pushed with commit typo fixed)

### CODE REVIEW

File: src/Data/Order/OrderDataClass.ts [hunk 1]
```diff
@@ -189,23 +154,18 @@ export const Order = provideLocalStorageDataClass('Order', {
...
       salesPartner: '',
       status: 'PSA_PRO_ORD_CLS',
-      statusLocalized: 'order completed',
       mainStatus: 'PSA_SAP_MAI_QUO_ORD_PRP',
-      mainStatusLocalized: 'Evaluation',
       open: false,
       quoteAt: null,
       orderAt: '2024-01-02T23:00:00Z',
       deliveryAt: '2023-07-23T13:35:38Z',
       termsOfDelivery: 'PSA_TOD_10',
```
Comment (apepper) [inline src/Data/Order/OrderDataClass.ts:167]: This is probably outdated. See https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-localize-order.scrivito-portal-app.pages.dev/order-details-75f5feaca1004671?order_id=F5F3792034404D40B41A843B61DC60D1&_scrivito_workspace_id=i6c8d84ddc5950c2&_scrivito_display_mode=diff

Comment (apepper) [reply src/Data/Order/OrderDataClass.ts:167]: Pisa still answers with the following data (e.g. `GET /order/F5F3792034404D40B41A843B61DC60D1`):

```json
    "termsOfDelivery": "PSA_TOD_10",
    "termsOfDeliveryLocalized": "",
```

Please clarify.

**Review by apepper — CHANGES_REQUESTED** [review]
FEEDBACK ([0eff931](https://github.com/Scrivito/scrivito-portal-app/pull/339/commits/0eff931fd2db8bbc87e0df877ea3c405abf62a84))

Almost there! See comments.

File: src/Data/Order/OrderDataClass.ts (hunk 1 above)
Comment (dcsaszar) [reply src/Data/Order/OrderDataClass.ts]: Clarified.

**Pull request merged**